tests/testthat/test-xamples.R

```diff
@@ -5,6 +5,22 @@ unit_conversions <- traits.build:::get_unit_conversions("config/unit_conversions
 taxon_list <- read_csv_char("config/taxon_list.csv")
 examples_dir <- "examples"
 
+# Dataset test
+expect_no_error(
+  dataset_test(
+    dataset_ids = c("Test_2023_2", "Test_2023_3", "Test_2023_4", 
+                    "Test_2023_5", "Test_2023_6", "Test_2023_7"), 
+    path_data = "examples")
+  )
+
+# Test_2023_1 explicitly has an incorrect trait_name included in the metadata file, so will fail dataset_test
+testthat::expect_output(
```

Comment:
I first tried to use "expect_error" and it doesn't work - because the function runs - so it isn't an error. It is just that the function documents "internal errors". It took a while to work out which "expect_" function did properly document this output and `expect_output` was the only one that worked. Maybe you know a different syntax to use with `expect_error`